InternalDocs/garbage_collector.md

```diff
@@ -16,7 +16,7 @@ count field can be examined using the `sys.getrefcount()` function (notice that
 value returned by this function is always 1 more as the function also has a reference
 to the object when called):
 
-```pycon
+```python
```

Comment:
I don't think it's correct.  Below is a console session example.

Also beware, there is https://github.com/python/cpython/pull/126887